.travis.yml

```diff
@@ -23,7 +23,7 @@ before_install:
   - python -m pip install pipenv
 
 install:
-  - pipenv install --dev
+  - pipenv install --ignore-pipfile --dev
```

Comment:
As per [this issue](https://github.com/pypa/pipenv/issues/3150) I think you want to be using `pipenv sync` instead?

Reply:
Ah, ok. All I want is for Travis to install exactly the versions from pipfile.lock. So it sounds like `sync` is the right thing